src/js_v2/presentation/registerCar.js

```diff
@@ -0,0 +1,8 @@
+import Car from '../domain/Car.js';
+
+export default function registerCar(carName) {
+  if(Array.isArray(carName)) return;
+  
+  return carName.split(',')
+                .map(it => new Car(it.trim()));
```

Comment:
```suggestion
                .map(Car.of);
```
이렇게 사용할 수 있는 static method를 하나 만들어주면 어떨까요?